### PR TITLE
Add deleteInt command, update UG and HelpWindow

### PR DESCRIPTION
**Pull request by ElijahS67**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteInterestCommand.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.interest.Interest;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds a list of interests to the specified person.
```
Comment (sltsheryl) [inline src/main/java/seedu/address/logic/commands/DeleteInterestCommand.java:17]: Deletes

File: src/main/java/seedu/address/logic/parser/DeleteInterestCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import java.util.stream.Collectors;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.DeleteInterestCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.interest.Interest;
+
+
+/**
+ * Parses input arguments and creates a new AddInterestCommand object
```
Comment (sltsheryl) [inline src/main/java/seedu/address/logic/parser/DeleteInterestCommandParser.java:20]: DeleteInterestCommand

**Review by sltsheryl — COMMENTED** [review]
JavaDocs has some errors

**Comment by sltsheryl** [comment]
Could also add ```deleteInt``` under the Features section of UG

File: src/main/java/seedu/address/logic/commands/DeleteInterestCommand.java (hunk 1 above)
Comment (ElijahS67) [reply src/main/java/seedu/address/logic/commands/DeleteInterestCommand.java:17]: Sorry about that, seemed to miss out some

**Review by carriezhengjr — APPROVED** [review]
LGTM. 👍

**Pull request merged**